kubernetes/samples/charts/util/setup.sh

```diff
@@ -5,6 +5,8 @@
 #  This script is to create or delete Ingress controllers. We support two ingress controllers: traefik and voyager.
 
 MYDIR="$(dirname "$(readlink -f "$0")")"
+VNAME=operator-v  # release name of Voyager
```

Comment:
What is the name of the generated service for Voyager operator after this change? voyager-operator-v?
And is it still traefik-operator for Traefik?

Reply:
With this change, install voyager via setup.sh:
```
$ kubectl -n voyager get all
NAME                                      READY     STATUS    RESTARTS   AGE
pod/voyager-operator-v-6fbd684b97-q7sf5   1/1       Running   0          56s

NAME                         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
service/voyager-operator-v   ClusterIP   10.103.72.200   <none>        443/TCP,56791/TCP   56s

NAME                                 DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/voyager-operator-v   1         1         1            1           56s

NAME                                            DESIRED   CURRENT   READY     AGE
replicaset.apps/voyager-operator-v-6fbd684b97   1         1         1         56s
```

With this change, install traefik via setup.sh:
```
$ kubectl -n traefik get all
NAME                                      READY     STATUS    RESTARTS   AGE
pod/traefik-controller-56d9554485-pdvm6   1/1       Running   0          7d

NAME                                   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
service/traefik-controller             NodePort    10.107.235.71   <none>        80:30305/TCP,443:30443/TCP   7d
service/traefik-controller-dashboard   ClusterIP   10.103.77.77    <none>        80/TCP                       7d

NAME                                 DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/traefik-controller   1         1         1            1           7d

NAME                                            DESIRED   CURRENT   READY     AGE
replicaset.apps/traefik-controller-56d9554485   1         1         1         7d
```

Reply:
we still wants the release name has sort of identifier

Reply:
What about the resources that contain the TNAME (traefik-operator)? I am trying to see if we could make the resource names for voyager and traefik consistent.

Reply:
the resource names are controlled by the specific helm chart and I didn't find a way to customize it. For Voyager, the names have pattern like 'voyager-<release-name>xxx' while for Traefik the names have pattern like 'traefik-controllerxxx'.

Reply:
Thanks for the explanation. I personally vote that we keep the names as they are. Firstly, in the existing version, at least, the release names (which are what the samples have control) are consistent. Secondly, the official example in the Voyager helm site doc uses the same release name ("voyager-operator") as what the samples currently have. 

We need more reviewers to chime in on this.

Reply:
As to the release name itself I also prefer "voyager-operator". The downside is the generated resource names will have redundant words as QA indicated. But since we know how the resource name is generated it can be explained. And since the release name is now parameterized, QA can change it if they want. I'll change back the Voyager release name.

Reply:
Can we add something in the doc to explain?

Reply:
Good idea! Will do.